.github/workflows/release.yml

```diff
@@ -20,12 +20,12 @@ jobs:
     steps:
       - uses: RDXWorks-actions/checkout@main
 
-      - name: Install Rust Toolchain for x86_64-apple-ios
+      - name: Install Rust Toolchain for aarch64-apple-darwin
         uses: RDXWorks-actions/toolchain@master
         with:
           toolchain: ${{ env.RUST_TOOLCHAIN }}
           components: ${{ env.RUST_COMPONENTS }}
-          target: x86_64-apple-ios
+          target: aarch64-apple-darwin
 
```

Comment:
We don't need x86 iPhones, they are far too old to even run iOS 16.

But we need macOS- Apple Silicon only- so change target to macOS.